sdk/eventgrid/azure-eventgrid/README.md

```diff
@@ -60,10 +60,7 @@ Information about the key concepts on Event Grid, see [Concepts in Azure Event G
 
 ### EventGridPublisherClient
 `EventGridPublisherClient` provides operations to send event data to topic hostname specified during client initialization.
-Either a list or a single instance of CloudEvent/EventGridEvent/CustomEvent can be sent.
-
-### EventGridDeserializer
-`EventGridDeserializer` is used to desrialize an event received.
+Either a list or a single instance of CloudEvent/EventGridEvent can be sent. Alternatively, a list or a single instance of the dict representation of CloudEvent/EventGridEvent or a Custom Schema can also be published.
```

Comment:
The wording is a little confusing. Might be more clear to separate the idea that CloudEvent/EGEvents can be sent as dicts and that events in a custom schema can be sent as a dict or list of dicts. Would be useful to add links for either the samples of sending all these different formats or from the `champion_scenarios.md` . Something like?

```suggestion
The following formats of events are allowed to be sent:
- a [list of strongly-typed EventGridEvents](add link to list of strongly typed EGE from champion_scenarios.md) 
- a [list of strongly-typed CloudEvents](add link to scenario from champion_scenarios.md) 
- a [single strongly-typed EventGridEvent instance](add link to single strongly typed EGE from champion_scenarios.md) 
- a [single strongly-typed CloudEvent instance](add link to scenario from champion_scenarios.md)
- a list of the [dict representation of EventGridEvents](add link) 
- a list of the [dict representation of CloudEvents](add link)
- a single instance of the [dict representation of EventGridEvent](url)
- a single instance of the [dict representation of CloudEvent](url)
- a list of [dicts in a custom schema](link)
- a single [dict in a custom schema](url)
```

Reply:
it would be worth it - i'll tackle it in the samples issue though since i fell this is a non blocker. changing the wording a bit for now